src/caf/toolkit/config_base.py

```diff
@@ -220,6 +220,39 @@ def write_example(
 
 
 # # # FUNCTIONS # # #
+def _is_collection(object: any):
+    """
+    Check if an object is any type of non-dict collection.
+
+    Currently only checks for list, tuple or set,
+    """
+    if isinstance(object, list):
+        return True
+    elif isinstance(object, tuple):
+        return True
+    elif isinstance(object, set):
+        return True
+    elif isinstance(object, dict):
+        return True
+    return False
```

Comment:
Can take a tuple of types
```suggestion
return isinstance(object, (list, tuple, set, dict))
```